components/colorarea/metadata/colorarea.yml

```diff
@@ -24,12 +24,6 @@ examples:
 
         <div class="spectrum-ColorHandle spectrum-ColorArea-handle">
           <div class="spectrum-ColorHandle-color" style="background-color: rgb(255, 0, 0)"></div>
-          <svg class="spectrum-ColorLoupe">
```

Comment:
We had this code here in ColorArea and[ ColorWheel ](components/colorwheel/metadata/colorwheel.yml)which both use ColorHandle and in turn can use ColorLoupe. However they did not have the `is-open` class so the ColorLoupe was not displayed and when toggled to be visible it does not display correctly in its current state (does not include path for express and does not handle selected color). 

I opted to remove it instead of adding it since there is demo of ColorLoupe within ColorHandle and ColorSlider and adding the new code here would increase the lines for each of these .yml files significantly (~550 lines).